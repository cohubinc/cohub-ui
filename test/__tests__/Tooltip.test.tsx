import React from "react";
import { mount } from "enzyme";

import { Tooltip } from "dist";

it("renders without crashing", async () => {
  const wrapper = mount(
    <Tooltip content="test content">
      <div>Tooltip reference</div>
    </Tooltip>
  );

  expect(wrapper).toMatchSnapshot();
});
