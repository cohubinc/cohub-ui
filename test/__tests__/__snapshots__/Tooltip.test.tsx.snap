// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Tooltip
  arrow={true}
  content="test content"
  delay={
    Array [
      100,
      50,
    ]
  }
  duration={250}
  interactive={false}
  placement="top"
  theme="dark"
  trigger="mouseenter"
>
  <ForwardRef(TippyWrapper)
    animateFill={false}
    arrow={true}
    content="test content"
    delay={
      Array [
        100,
        50,
      ]
    }
    duration={250}
    interactive={false}
    placement="top"
    popperOptions={
      Object {
        "modifiers": Object {
          "preventOverflow": Object {
            "boundariesElement": "window",
          },
        },
      }
    }
    theme="dark"
    trigger="mouseenter"
  >
    <Tippy
      animateFill={false}
      arrow={true}
      content="test content"
      delay={
        Array [
          100,
          50,
        ]
      }
      duration={250}
      interactive={false}
      placement="top"
      popperOptions={
        Object {
          "modifiers": Object {
            "preventOverflow": Object {
              "boundariesElement": "window",
            },
          },
        }
      }
      theme="dark"
      trigger="mouseenter"
    >
      <span>
        <div>
          Tooltip reference
        </div>
      </span>
      <Portal
        containerInfo={
          <div>
            test content
          </div>
        }
      >
        test content
      </Portal>
    </Tippy>
  </ForwardRef(TippyWrapper)>
</Tooltip>
`;
